src/sentry/web/api.py

```diff
@@ -344,6 +346,23 @@ def process(self, request, project, key, auth, helper, data, **kwargs):
                     (tsdb.models.key_total_blacklisted, key.id),
                 ]
             )
+            if should_filter[1] == 'ip_address':
```

Comment:
I'd change this to be a dictionary lookup.

Something like:

```python
keys_to_models = {
  'ip_address': tsdb.models.project_total_received_ip_address,
  ...
}
```

Then the lookup here is simply:

```python
try:
    tsdb.incr(keys_to_models[should_filter[1], project.id)
except KeyError:
    pass
```